src/common/applicationDetails/ApplicationDetails.tsx

```diff
@@ -49,6 +49,14 @@ interface BerthSwitch {
   reason: string | null;
 }
 
+interface WinterStorageAreaSummary {
+  applicationCode: string;
+  acceptBoatingNewsletter: boolean;
+  acceptFitnessNews: boolean;
+  acceptLibraryNews: boolean;
+  acceptOtherCultureNews: boolean;
```

Comment:
hm, but these are common for both winter storage and berth applications 🤔

Reply:
Oh, that's right. Incorrectly assumed they didn't exist on berth applications as the component did not already include them. Will also query those for berth applications and rename the key.

Reply:
Done.